pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/plugins/ServiceBusProcessor.kt

```diff
@@ -39,8 +46,10 @@ class ServiceBusProcessor {
      */
     @Throws(BadRequestException::class)
     fun withMessage(message: ServiceBusReceivedMessage) {
+        validateJsonSchema(message)
         val sbMessageId = message.messageId
-        var sbMessage = message.body.toString()
+        //var sbMessage = message.body.toString()
+        var sbMessage =String(message.body.toBytes())
         val sbMessageStatus = message.state.name
         try {
             logger.info { "Before Message received = $sbMessage" }
```

Comment:
Please update Failed to parse CreateReportSBMessage: ${e.localizedMessage} message.  It's no longer CreateReportSBMessage.